docs/releases/changelog-dev.md

```diff
@@ -57,6 +57,27 @@
 
   [#762](https://github.com/qilimanjaro-tech/qililab/pull/762)
 
+- Added `filter` argument inside the qua config file compilation from runcards with qm clusters. This is an optional element for distorsion filters that includes feedforward and feedback, two distorion lists for distorsion compensation and fields in qua config filter. These filters are calibrated and then introduced as compensation for the distorsions of the pulses from external sources such as Bias T. The runcard now might include the new filters (optional):
+
+  ```
```

Comment:
I liked this Example thing here oyu added on the other PR, maybe add it here aswell?
```suggestion
  Example:
    ```
```
Notice you will need to tab the rest of the Example too!